cache.py

```diff
@@ -24,8 +27,13 @@ def load(self, pid, lba_base, nblks):
             if lba in self.pool:
                 self.policy.hit(lba)
                 self.n_hits += 1
+                if lba in self.prefetched:
```

Comment:
when cache hit occurs, if the address is cached by prefetching then increase n_prefetch hit.